addons/mail/static/src/components/discuss_sidebar_mailbox/discuss_sidebar_mailbox.scss

```diff
@@ -1,57 +1,5 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
 .o_DiscussSidebarMailbox {
-    display: flex;
-    align-items: center;
-    padding: map-get($spacers, 1) 0;
-}
-
-
-.o_DiscussSidebarMailbox_item {
-    margin-left: $o-mail-discuss-sidebar-category-item-margin;
-    margin-right: $o-mail-discuss-sidebar-category-item-margin;
-
-    &:first-child {
-        margin-left: $o-mail-discuss-sidebar-category-item-avatar-left-margin;
-    }
-
-    &:last-child {
-        margin-right: $o-mail-discuss-sidebar-scrollbar-width;
-    }
-}
-
-
-.o_DiscussSidebarMailbox_name {
-    @include text-truncate();
-}
-
-// ------------------------------------------------------------------
-// Style
-// ------------------------------------------------------------------
-
-.o_DiscussSidebarMailbox {
-    cursor: pointer;
-
-    &.o-active {
-        background-color: gray('200')
-    }
-
     &:hover {
-        background-color: gray('300');
+        background-color: gray('300') !important;
     }
-
-    &.o-starred-box {
-        .o_DiscussSidebarMailbox_counter {
-            border-color: gray('400');
-            background-color: gray('400');
-            color: gray('300');
-        }
-    }
-}
-
-.o_DiscussSidebarMailbox_counter {
-    background-color: $o-brand-primary;
```

Comment:
There are too many differences on badges with master:
![mailbox](https://user-images.githubusercontent.com/80679690/166425456-bda15f93-054d-44a2-9277-2b4485a91492.jpg)

What do you think about using `badge-primary` and `bg-400` on the related badges?

Reply:
My bad, I forgot about `Inbox !== Starred` in terms of style.

Indeed, using `badge-primary` and `bg-400` would work here. Additionally, I have added a `text-light` in the `bg-400` user case to make sure that the text remains light in every situation.